app/controllers/user.js

```diff
@@ -0,0 +1,3 @@
+const errors = require('../errors');
```

Comment:
I meant the entire user controllers, why are you adding this file to the repo if you are not using it yet?